projects/spectator/tsconfig.lib.json

```diff
@@ -6,10 +6,10 @@
     "declaration": true,
     "inlineSources": true,
     "typeRoots": [
+      "../../node_modules/@types",
```

Comment:
Can you explain this change, please?

Reply:
typescript got upgraded from `5.0.4` to `5.2.2`

Typescript 5.1 introduced a change in `typeRoots` behaviour:
> Explicit `typeRoots` Disables Upward Walks for `node_modules/@types`

More info: https://devblogs.microsoft.com/typescript/announcing-typescript-5-1/

Reply:
Looking at the Microsoft documentation the order of `typeRoots` should be corrected:

```
    "typeRoots": [
      "./typings",
      "../../node_modules/@types"
    ],
```